scripts/bench/README.md

```diff
@@ -2,11 +2,21 @@ Work-in-progress benchmarks.
 
 ## Running the suite
 
+You'll need two folders to compare, each of them containing `react.min.js` and `react-dom-server.min.js`. You can run `npm run build` at the repo root to get a `build` folder with these files.
```

Comment:
It would be nice if there was an easier way to run against a release build without manually recreating the build folder.